src/nvim/terminal.c

```diff
@@ -201,7 +201,7 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
-  vterm_state_set_bold_highbright(state, true);
+  vterm_state_set_bold_highbright(state, false);
```

Comment:
Yeah we could, since as a default libvterm initializes the state of this variable to false. I just thought about keeping it in case libvterm changes its defaults (unlikely). Let me know what you prefer :)